ui/common/StatusBar.tsx

```diff
@@ -59,7 +59,7 @@ export default function StatusBar({
 
   return (
     <div
-      className={clsx('border-t border-white/25 max-md:px-4 max-md:py-8', {
+      className={clsx('border-t border-white/25 max-md:px-4 max-md:py-8 max-md:bottom-0 max-md:fixed', {
```

Comment:
for reference the fixed position will keep the statusbar at the bottom so when they keyboard is opened it will dissappear in this PR currently